deploy/crds/openebs_v1alpha1_openebsinstalltemplate_crd.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: openebsinstalltemplates.openebs.io
+spec:
+  group: openebs.io
+  names:
+    kind: OpenEBSInstallTemplate
+    listKind: OpenEBSInstallTemplateList
+    plural: openebsinstalltemplates
+    singular: openebsinstalltemplate
+  scope: Namespaced
+  subresources:
+    status: {}
+  version: v1alpha1
+  versions:
```

Comment:
Do these flags require a minimum k8s version where they will work?

Reply:
The CRD itself was applied/tested on 1.11, 1.12 and 1.13 (GKE/Packet & OpenShift 4.1) @kmova ..
However, there is a `minKube` version that is mandatory to be specified in the OLM CR. Ex: https://github.com/openebs/helm-operator/blob/bb4028aa812d9ea34856d5c89c5626091b90520f/olm/openebsoperator.v1.0.0.clusterserviceversion.yaml#L209

Here, we have provided 1.12 considering the support for LocalPV.